src/LCM2ROSControl.hpp

```diff
@@ -40,6 +44,14 @@ namespace valkyrie_translator
     double ff_qd_d;
     double ff_f_d;
     double ff_const;
+  } joint_gains;
```

Comment:
good point. I was just trying to match the style of the rest of the file, but I'll change them both to the c++ way